src/main/java/uk/ac/cam/cl/dtg/segue/dao/content/ContentMapper.java

```diff
@@ -145,43 +123,6 @@ public Content load(final String docJson) throws IOException {
         }
     }
 
-    /**
-     * Register a new content type with the content mapper.
-     * 
-     * @param type
-     *            - String that should match the type field of the content object.
-     * @param cls
-     *            - Class implementing the deserialized DO.
-     */
-    public synchronized void registerJsonTypeToDO(final String type, final Class<? extends Content> cls) {
-        Validate.notEmpty(type, "Invalid type string entered. It cannot be empty.");
-        Validate.notNull(cls, "Class cannot be null.");
-
-        jsonTypes.put(type, cls);
-    }
-
-    /**
-     * Works the same as {@link #registerJsonTypeToDO(String, Class)}.
-     * 
-     * @see #registerJsonTypeToDO(String, Class)
-     * @param newTypes
-     *            a map of types to merge with the segue type map. The classes added to the map must contain jsonType
-     *            annotations and may contain DTOMapping annotations.
-     */
-    public synchronized void registerJsonTypes(final List<Class<? extends Content>> newTypes) {
```

Comment:
These methods allowed changing the set of known types later by adding in new ones to be mapped. This was unreliable and maybe the cause of the mapping errors. Since we now initialise `segue` and `isaac` mappings simultaneously by treating it as one codebase, we don't _need_ to add more types later and this possibly-dodgy functionality can be removed.